specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -2978,6 +2996,10 @@
             "source",
             "target"
           ]
+        },
+        "kernel": {
```

Comment:
>kernel [](http://example.com/codeflow?start=9&length=6)

Consider naming this something more descriptive/specific (jupyterConfig, jupyterKernelConfig,or similar/better).

Reply:
We will consider renaming this to jupyterKernelConfig. 

@Ada, Please consider making changes to the C3 accordingly.

Reply:
we are leaving as kernel. this value has been in production for quite a while, we are just adding documentation. due to the design changing the name of values like this causes issues because it is stateful on customer controlled compute.

Reply:
Had a discussion on this, currently only jupyter is supported but we will plan for other kernels too so in line we wish to keep the name same.